DataWatcher

is everything ok with 1.13.2

```
getting a ton of errors about Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType
at org.inventivetalent.glow.GlowAPI.sendGlowPacket(GlowAPI.java:293) ~[?:?]
```

```
Caused by: java.lang.ExceptionInInitializerError
at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<init>(DataWatcher.java:330) ~[?:?]
at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<clinit>(DataWatcher.java:202) ~[?:?]
at org.inventivetalent.glow.GlowAPI.sendGlowPacket(GlowAPI.java:293) ~[?:?]
at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:116) ~[?:?]
at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:133) ~[?:?]
```

PacketListenerAPI_v3.7.1-SNAPSHOT.jar
GlowAPI_v1.4.7-SNAPSHOT.jar

```
<dependency>
      <groupId>org.inventivetalent</groupId>
      <artifactId>glowapi</artifactId>
      <version>1.4.7-SNAPSHOT</version>
    </dependency>
```

Latest 1.12.2 spigot release
had to disable this for now since i cant catch an ExceptionInInitializerError

```
[19:05:06 ERROR]: Could not pass event PlayerSwapHandItemsEvent to Solinia3Core v1.0
 
org.bukkit.event.EventException: null
 
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:309) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1099) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:40) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.PacketPlayInBlockDig.a(SourceFile:10) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
 
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_111]
 
at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:895) [spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:830) [spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:728) [spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
 
Caused by: java.lang.ExceptionInInitializerError
 
at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<init>(DataWatcher.java:330) ~[?:?]
 
at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<clinit>(DataWatcher.java:202) ~[?:?]
 
at org.inventivetalent.glow.GlowAPI.sendGlowPacket(GlowAPI.java:293) ~[?:?]
 
at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:116) ~[?:?]
 
at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:133) ~[?:?]
 
at com.solinia.solinia.Utils.Utils.setGlowing(Utils.java:7039) ~[?:?]
 
at com.solinia.solinia.Managers.EntityManager.setEntityTarget(EntityManager.java:1314) ~[?:?]
 
at com.solinia.solinia.Listeners.Solinia3CorePlayerListener.onPlayerSwapHandItems(Solinia3CorePlayerListener.java:460) ~[?:?]
 
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
 
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
 
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
 
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]        at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<init>(DataWatcher.java:330) ~[?:?]
 
        at org.inventivetalent.glow.reflection.minecraft.DataWatcher$V1_9$ValueType.<clinit>(DataWatcher.java:202) ~[?:?]
 
        at org.inventivetalent.glow.GlowAPI.sendGlowPacket(GlowAPI.java:293) ~[?:?]
 
        at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:116) ~[?:?]
 
        at org.inventivetalent.glow.GlowAPI.setGlowing(GlowAPI.java:133) ~[?:?]
 
        at com.solinia.solinia.Utils.Utils.setGlowing(Utils.java:7039) ~[?:?]
 
        at com.solinia.solinia.Managers.EntityManager.setEntityTarget(EntityManager.java:1314) ~[?:?]
 
        at com.solinia.solinia.Listeners.Solinia3CorePlayerListener.onPlayerSwapHandItems(Solinia3CorePlayerListener.java:460) ~[?:?]
 
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
 
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
 
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
 
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
 
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[spigot-1.13.2.jar:git-Spigot-518206a-917411f]
 
        ... 15 more
```

--- a/pom.xml
+++ b/pom.xml
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>org.inventivetalent</groupId>
             <artifactId>reflectionhelper</artifactId>
-            <version>1.14.0-SNAPSHOT</version>
+            <version>1.14.3-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.inventivetalent</groupId>

